bootstrap.sh

```
@@ -1 +1,9 @@
 #!/bin/bash
+kubectl apply -f .infrastructure/namespace.yml
+kubectl apply -f .infrastructure/pv.yml
+kubectl get pv todoapp-pv 
+kubectl apply -f .infrastructure/pvc.yml
+kubectl apply -f .infrastructure/deployment.yml
+kubectl apply -f .infrastructure/service.yml
+kubectl get svc -n todoapp
+echo
```

Empty `echo` provides no information. Replace with a meaningful completion message such as `echo "Bootstrap complete. Run kubectl get pods -n todoapp to check pod status."`. Also add `set -e` at the top of the script to fail fast on errors and ensure the script is executable (`chmod +x bootstrap.sh`).